backends/kv/backend.go

```diff
@@ -123,5 +122,15 @@ func (b *Backend) newConnection(session *frames.Session, password string, token
 		token,
 		b.logger)
 
+	if err == nil && b.numWorkers == 0 {
```

Comment:
Wouldn't it make more sense to do this earlier, when we create the backend or the v3io session?

Reply:
I suppose the downside(?) would be that it would require cluster connectiity to start frames.

Reply:
i need user pass or token to open a connection which i don't have on startup, need to wait for a req and use it from there